api/CampusService.proto

```diff
@@ -85,36 +86,214 @@ service Campus {
+    };
+  }
+
+
+  rpc GetResponsiblePerson (google.protobuf.Empty) returns (GetResponsiblePersonReply) {
+    option (google.api.http) = {
+      get: "/barrierfree/contacts",
+    };
+  }
+
+  rpc GetBuilding2Gps (google.protobuf.Empty) returns (GetBuilding2GpsReply) {
+    option (google.api.http) = {
+      get: "/barrierfree/getBuilding2Gps",
+    };
+  }
+
+  rpc GetAreaFacilitiesByBuildingNr (GetAreaFacilitiesByBuildingNrRequest) returns (GetAreaFacilitiesByBuildingNrReply) {
+    option (google.api.http) = {
+      get: "/barrierfree/nerby",
+    };
+  }
+
+  rpc GetListOfToilets (google.protobuf.Empty) returns (GetListOfToiletsReply) {
+    option (google.api.http) = {
+      get: "/barrierfree/listOfToilets",
+    };
+  }
+
+  rpc GetListOfElevators (google.protobuf.Empty) returns (GetListOfElevatorsReply) {
+    option (google.api.http) = {
+      get: "/barrierfree/listOfElevators",
+    };
+  }
+
+  rpc GetMoreInformation (google.protobuf.Empty) returns (GetMoreInformationReply) {
+    option (google.api.http) = {
+      get: "/barrierfree/moreInformation",
+    };
+  }
+
+
+  rpc GetOpeningTimes (GetOpeningTimesRequest) returns (GetOpeningTimesReply) {
+    option (google.api.http) = {
+      get: "/openingtimes/{language}",
+    };
+  }
+
+
+  rpc GetUpdateNote (GetUpdateNoteRequest) returns (GetUpdateNoteReply) {
+    option (google.api.http) = {
+      get: "/updatenote/{version}",
+    };
+  }
+
+
+  rpc GetStudyRoomList (google.protobuf.Empty) returns (GetStudyRoomListReply) {
+    option (google.api.http) = {
+      get: "/studyroom/list",
+    };
+  }
+
+  rpc GetEventList(GetEventListRequest) returns (GetEventListReply) {
+    option (google.api.http) = {
+      get: "/event/list",
+    };
+  }
+
+  rpc GetPurchasedTickets(google.protobuf.Empty) returns (GetPurchasedTicketsReply) {
+    option (google.api.http) = {
+      post: "/event/ticket/my",
+    };
+  }
+
+  rpc GetMyTicket(GetTicketRequest) returns (EventListMsgElement) {
+    option (google.api.http) = {
+      post: "/event/ticket/{id}",
+    };
+  }
+
+  rpc GetTicketType(GetTicketRequest) returns (GetEventTicketTypeReply) {
+    option (google.api.http) = {
+      get: "/event/ticket/type/{id}",
+    };
+  }
+
+  rpc ReserveMultipleTickets(ReserveMultipleTicketsRequest) returns (ReserveMultipleTicketsReply) {
+    option (google.api.http) = {
+      post: "/event/ticket/reserve/multiple",
+    };
+  }
+
+  rpc PurchaseTicketStripe(google.protobuf.Empty) returns (PurchaseTicketStripeReply) {
+    option (google.api.http) = {
+      post: "/event/ticket/payment/stripe/purchase/multiple",
+    };
+  }
+
+  rpc GetEphemeralKey(google.protobuf.Empty) returns (GetEphemeralKeyReply) {
+    option (google.api.http) = {
+      post: "/event/ticket/payment/stripe/ephemeralkey",
+    };
+  }
+
+  rpc GetTicketStatus(GetTicketStatusRequest) returns (GetTicketStatusReply) {
+    option (google.api.http) = {
+      get: "/event/ticket/status/{eventId}",
+    };
+  }
```

Comment:
Let's do this ticket stuff some other time, i don't think they'll be used currently.
```suggestion
```